db.php

```diff
@@ -289,8 +289,7 @@ function chooseMentor() {
 
 
 	 function getMentor($mentor) {
-		global $_USER;
-		$user = $_USER['uid'];
+		$user = $mentor;
```

Comment:
This means that anyone can access anyone's mentor, right? Is that ok?

If that's a problem, we should probably just make a note that it needs to be fixed.